writings/2022-09-12-order-flow-auctions-and-centralisation-1.mdx

```diff
@@ -0,0 +1,96 @@
+
+### A good MEV market
+
+The particular MEV market in question in this article is [PBS](https://ethresear.ch/t/proposer-block-builder-separation-friendly-fee-market-designs/9725) in the form of [MEV-Boost](https://boost.flashbots.net/), but many different forms of MEV markets likely succumb to similar descriptions and dangers. In order to argue that EOF can have undesirable effects on the MEV market, it is important to first establish some of the properties we require from PBS:
+
+- **(Approximately) dominant strategy for proposers:** This is the most essential aim of PBS, so hopefully you are already familiar with this idea. At a high level, the aim is that either the proposer’s best strategy is to build blocks through the builder market or their best strategy yields only marginally higher returns than what the builder market has to offer. We have this goal to prevent centralisation of stake on Ethereum.
+- **No rent-seeking:** The majority of the MEV that is extracted should not remain with the extracting parties (searchers/builders), but should rather flow back to the rest of the ecosystem. Where exactly the value should flow is a topic for another article, but the obvious contenders are validators, users and DApps.
+- **Censorship resistance:** just like before PBS, a valid order should be included eventually and, ideally, without much delay.
+- **Improvements to UX:** admittedly this is more of a desired than a required feature. However, given the addition of a sophisticated agent to the Ethereum ecosystem, it seems remiss not to leverage this agent for improvements to areas of the ecosystem we know to be weak. More detail is added to what UX improvements are possible.
+
+### Where the orders flow
+
+Consider a wallet trying to send exclusive order flow to a single builder. In order for the wallet’s orders to be executed the builder who receives the orders must land a block on chain, which may not happen immediately. End users don’t like delays and delays also make things like gas fee estimation a lot harder so the wallet is incentivised to minimise this delay by sending OF to the builder with the highest inclusion rate, further increasing their dominance of the market. Clearly, this is a centralising force on the builder market.
+
+There are various reasons why a wallet might do this. The most obvious is contractual payment for order flow from the builder, but it might also be in exchange for certain features from the builder or because the wallet maintainer is too lazy to integrate with any builder apart from the largest, dominant builder that produces most blocks anyway. These motives and the ability to control order flow are not unique to wallets. For instance, DApp UI’s are in a very similar position. Lets call an agent that could play this role an “originator” for generality.
+
+![Untitled](/img/OF1.png)
```

Comment:
instead of "Untitled", consider adding some alt tag (for people with impaired vision)

Reply:
Done